Sdk/MoyasarSdk/Services/Models/ApiSTCPaySource.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ApiSTCPaySource.swift
+//  MoyasarSdk
+//
+//  Created by Mahmoud Abdelwahab on 17/09/2024.
+//
+
+import Foundation
+
+// MARK: - ApiSTCPaySource
+public struct ApiSTCPaySource: Codable {
+    public let type, mobile, referenceNumber, branch: String?
```

Comment:
Bro  it is recommended to make the codable object as optional to avoid any un expected response so i safe unwrap it like null safty in dart 
- Please initialize type with hardcoded value stcpay--->   this is done while creating the object because i reuse it in request and response 
- Please define each attribute in a single line  ---> Done

Reply:
I see your point. Don't worry about it then. 👌